src/Tooltip.js

```diff
@@ -247,6 +248,7 @@ class Tooltip extends React.Component {
         target={this.props.target}
         isOpen={this.props.isOpen}
         hideArrow={this.props.hideArrow}
+        boundariesElement={this.props.boundariesElement}
```

Comment:
Can you also make a similar change for the `Popover` component?

Reply:
👍 done

Reply:
👍 done